settings

```diff
@@ -51,9 +51,8 @@ KEYDIR="$RELEASEDIR/gnupg"
 GNUPGHOME="$KEYDIR"
 TARDIR="$RELEASEDIR/tarballs"
 EXTRASDIR="$RELEASEDIR/extras"
-HOSTS="web01.osuosl.theforeman.org,"
 DOWNLOADS_HOSTS="website01.osuosl.theforeman.org,"
-YUM_HOSTS="${HOSTS}"
+YUM_HOST="web01.osuosl.theforeman.org"
```

Comment:
RPM_HOST? or save that for a later change?

Reply:
sure, if you want :) (done)